src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -180,9 +180,9 @@ const { resource, id } = Astro.params;
 ```
 This page will be served for any value of `resource` and `id`: `resources/users/1`, `resources/colors/blue`, etc.
 
-#### Modifying the `[...slug]` example for SSR
+#### Modifying the `[...slug]` example for on-demand rendering
 
-Because SSR pages can't use `getStaticPaths()`, they can't receive props. The [previous example](#example-dynamic-pages-at-multiple-levels) can be adapted for SSR mode by looking up the value of the `slug` param in an object. If the route is at the root ("/"), the slug param will be `undefined`. If the value doesn't exist in the object, we redirect to a 404 page.
+Because pages rendered on demand can't use `getStaticPaths()`, they can't receive props. The [previous example](#example-dynamic-pages-at-multiple-levels) can be adapted for `server` or `hybrid` mode by looking up the value of the `slug` param in an object. If the route is at the root ("/"), the slug param will be `undefined`. If the value doesn't exist in the object, we redirect to a 404 page.
```

Comment:
Same comment! Not quite sure how to talk about “server” or “hybrid + prerender = false” being equivalent

Reply:
Yes, agreed! It's not a perfect science, so we'll flag all the awkward cases. I do like "pages (or routes??) rendered on demand" because that covers our bases.

I've been alternating with talking about the "mode enabled", but yes, we should make extra sure that in a case like this, we're referencing what is actually happening for this exact page. I'm sure there will be several of these we'll refine as we figure out when we mean/need to say:

- you have configured this rendering mode
- the actual page/route in question is rendered on-demand